examples/sampleview/core/sampledesigner/ILayerView.cpp

```diff
@@ -69,108 +56,66 @@ void ILayerView::mousePressEvent(QGraphicsSceneMouseEvent *event)
     QGraphicsItem::mousePressEvent(event);
 }
 
+auto ILayerView::determineAction(const MultiLayerView *new_parent, int row)
+{
+    // Simple move of single layer on the scene
+    if (!new_parent && !parentItem())
```

Comment:
You probably scheduled this for later, but I will be glad to see soon all these 'if' statement replaced with functions like
`isSimpleMove(current_parent, new_parent, row)`